web/style/common.scss

```diff
@@ -0,0 +1,284 @@
+@mixin flex {
+  display: flex;
+}
+
+@mixin flex-row {
+  @include flex;
+  flex: 1;
+}
+
+@mixin flex-column {
+  @include flex;
+  flex: 1;
+  flex-direction: column;
+}
+
+@mixin flex-row-reverse {
+  @include flex;
+  flex-direction: row-reverse;
+}
+
+@mixin flex-column-reverse {
+  @include flex;
+  flex-direction: column-reverse;
+}
+
+@mixin break-point($point) {
+  @if $point == tablet {
+    @media only screen and (min-width: 480px) {
+      @content;
+    }
+  } @else if $point == desktop {
+    @media only screen and (min-width: 1024px) {
+      @content;
+    }
+  }
+}
+
+@mixin app-header-padding-changes {
+  @include break-point(desktop) {
+    padding: 0 $app-padding-horizontal;
+  }
+}
+
+.flex {
+  @include flex;
+}
+
+.flex-row {
+  @include flex-row;
+}
+
+// Common file 
+.flex-column {
+  @include flex-column;
+}
+
+.flex-row-reverse {
+  @include flex;
+  flex: 1;
+  flex-direction: row-reverse;
+}
+
+.flex-column-reverse {
+  @include flex;
+  flex: 1;
+  flex-direction: column-reverse;
+}
+
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+  // color: $text-color-primary;
+}
+
+a {
+  color: $text-color-primary;
+  text-decoration: none;
+}
+
+html,
+body {
+  margin: 0 auto;
+  font-family: "Dosis";
+  font-weight: $app-font-weight;
+  box-sizing: border-box;
+}
+
+.btn {

```

Comment:
Its not modular way of writing CSS, why this css is not under button.scss ?

Reply:
Done